pom.xml

```diff
@@ -464,6 +468,10 @@
     </repositories>
 
     <pluginRepositories>
+        <pluginRepository>
+            <id>gradle-dev-public</id>
+            <url>https://repo.grdev.net/artifactory/public</url>
+        </pluginRepository>
```

Comment:
Is it your intention to keep these after 1.21 is released?

Reply:
No. As soon as 1.21 of the Maven extension is released, I'll change the version number to 1.21 and remove this repository again.